extensions/eclipse-che-theia-remote-api/src/common/plugin-service-impl.ts

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * Copyright (c) 2018-2020 Red Hat, Inc.
+ * Copyright (c) 2021 Red Hat, Inc.
```

Comment:
AFAIK there is no implementation in API, it's just interfaces

Reply:
it should be just interfaces of services being different between che-server or devWorkspaces.
There is no need to have abstract classes.

Reply:
There is already some common code https://github.com/eclipse-che/che-theia/blob/main/extensions/eclipse-che-theia-remote-api/src/browser/oauth-utils.ts

If you prefer to have only interfaces in this package, then we need to add one for abstract classes and utins.